jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/AdministrationController.java

```diff
@@ -332,6 +322,16 @@ public ModelAndView getSpamProtectionPage(){
         return new ModelAndView("spamProtection").addObject("rules", ruleDtos);
     }
 
+    @RequestMapping(value = "/usersgroup/list", method = RequestMethod.GET)
```

Comment:
In Spam Rules we use `/` to get the list of resources, I'd follow those conventions. And we're looking for users here, not for usergroups, right? So I suppose it should be: `/user`